tokens/test/erc-1155/ERC1155RaribleUser.test.js

```diff
@@ -177,6 +177,80 @@ contract("ERC1155RaribleUser", accounts => {
     await checkCreators(tokenId, [minter]);
   });
 
+  it("mint and transfer with minter access control", async () => {
+    const minter = accounts[1];
+    let transferTo = accounts[2];
+
+    const tokenId = minter + "b00000000000000000000001";
+    const tokenURI = "//uri";
+    let supply = 5;
+    let mint = 2;
+
+    await expectThrow(
+      token.mintAndTransfer([tokenId, tokenURI, supply, creators([minter]), [], [zeroWord]], transferTo, mint, {from: minter})
+    );
+
+    await token.grantMinter(minter, {from: tokenOwner});
+    assert.equal(await token.isValidMinter(minter), true);
+    assert.equal(await token.isValidMinter(transferTo), false);
+
+    await token.mintAndTransfer([tokenId, tokenURI, supply, creators([minter]), [], [zeroWord]], transferTo, mint, {from: minter});
+		assert.equal(await token.uri(tokenId), "ipfs:/" + tokenURI);
```

Comment:
Line 198-200. In This test we don`t need any additional checking. I think this code  interfere the main test idea check 'minter access control'
Please review another two tests with 'minter access control' and delete unnecessary code

Reply:
the `assert.equal(await token.uri(tokenId), "ipfs:/" + tokenURI);` has been removed.

do you also want to remove the following:
```
assert.equal(await token.balanceOf(transferTo, tokenId), mint);
assert.equal(await token.balanceOf(minter, tokenId), 0);
```

They should not be needed if `mintAndTransfer` works as intended but they are here to make sure the minter access control didn't break something.

Reply:
Hi @NicolasMahe, thanks for good work!
Ok agree let's leave this 2 lines in test:
`assert.equal(await token.balanceOf(transferTo, tokenId), mint);
assert.equal(await token.balanceOf(minter, tokenId), 0);`